packages/expo-modules-core/ios/Core/Views/SwiftUI/SwiftUIChild.swift

```diff
@@ -12,13 +12,19 @@ extension ExpoSwiftUI {
    */
   public struct Child: SwiftUI.View, Identifiable {
     public let id: Int
+    #if os(iOS)
     public let view: UIView
+    #endif
+    #if os(macOS)
+    public let view: NSView
```

Comment:
I wonder why [this alias](https://github.com/expo/expo/blob/main/packages/expo-modules-core/ios/Platform.swift#L7) doesn't work 🤔